internal/storage/ext.go

```diff
@@ -27,7 +27,6 @@ type Instances interface {
 	// todo: remove after instances parameters migration is done
 	InsertWithoutEncryption(instance internal.Instance) error
```

Comment:
`*WithoutEncryption` methods were used in https://github.com/kyma-project/control-plane/pull/514 and removed in  https://github.com/kyma-project/control-plane/pull/592. Please consider removing all of the methods if they are not used or creating a followup.